teamB/YaeRin/services/postService.js

```diff
@@ -0,0 +1,35 @@
+const postDao = require("../models/postDao");
+
+const postUpCtrl = async (title, content, userId) => {
```

Comment:
```suggestion
const create = async (title, content, userId) => {
```
postService를 호출하는 파일에서는 아래와 같이 사용할 수 있겠습니다.
```javascript
postService.create(title, content, userId)
postService.insert(title, content, userId)

```

Reply:
넵!